### PR TITLE
fix(email): Correct branding in invite emails

### PR DESCRIPTION
**Pull request by joethreepwood**
## Problem

Corrects the old branding hog in the invite email, to a new one. 

## Changes

Before 
Screenshot 2023-03-06 at 10 49 32

This replaces that old fashioned hog with a new one by Lottie, via https://github.com/PostHog/posthog.com/issues/4707#issuecomment-1455792753

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

Didn't. YOLO.

### CODE REVIEW

**Review by pauldambra — COMMENTED** [review]
Is there a way to view it in context?

The height has gone from 174px to 713px (and width nearly doubled)... 

Is that constrained in use?

----

Looking in the emails folder I can't see any styles for `img` tags so I think this would be 713px square in the emails. That's pretty big on a small screen

Should it have something like 

```
<div class="text-center">
    <img 
        src="{% absolute_asset_url 'email/invite-email-welcome.png' %}" 
        style="height:200px;width:auto;" <!-- NB I didn't check this is valid syntax!!! -->
    />
</div>
```

**Comment by Twixes** [comment]
I think there _was_ actually a way: we seem to render email templates in backend CI, and we used to upload those renders as CI artifacts. However, I've only seen that in code and never actually used those renders + I just noticed the artifact upload step hasn't been working properly for a few months. Fixing this in #14563.

**Comment by joethreepwood** [comment]
I completely defer to you both on this, tbh 🤷

**Comment by pauldambra** [comment]
merged the change in so we can see the email... top class CI fixing @Twixes 🙌

**Comment by pauldambra** [comment]
and the tests don't run on this PR 🤣

File: posthog/tasks/test/test_email.py [hunk 1]
```diff
@@ -58,8 +58,9 @@ def test_send_invite(self, MockEmailMessage: MagicMock) -> None:
         send_invite(invite.id)
 
         assert len(mocked_email_messages) == 1
-        assert mocked_email_messages[0].send.call_count == 1
-        assert mocked_email_messages[0].html_body
+        the_message = mocked_email_messages[0]
```
Comment (Twixes) [inline posthog/tasks/test/test_email.py:61]: Hmm I'd say the better change to run backend CI here would be to update `ci-backend.yml` to run on any `frontend/public/email/` change

**Comment by pauldambra** [comment]
Screenshot 2023-03-06 at 22 11 04

We could trim vertical space off to shift Max up a little?

Thoughts @joethreepwood @lottiecoxon

**Review by pauldambra — APPROVED** [review]
assuming this is what you want it to look like then have a tick 😊

**Comment by pauldambra** [comment]
ofc if you just want to merge the updated branding and move on then go for it, otherwise happy to help with how to style 😊

**Comment by joethreepwood** [comment]
@lottiecoxon How would you feel if we cropped it down to match the old style, like this?

invite-email-welcome 09 23 (1)

**Comment by lottiecoxon** [comment]
I would swap it for this as this makes the crop look more intentional

builder-hog-05

**Comment by joethreepwood** [comment]
@lottiecoxon Is it possible to get a smaller version of this, with height approx 174px?

**Comment by lottiecoxon** [comment]
voila - but the quality is quite bad at this size 
builder-hog-05-small
maybe this sized down to fit would be better?
builder-hog-05-small

**Review by pauldambra — APPROVED** [review]
Screenshot 2023-04-01 at 22 20 13

🚢

**Comment by pauldambra** [comment]
@joethreepwood this is all good now, but I don't want to merge it and then go off on holiday 😊 

Feel free to merge this on Monday 👍

**Pull request merged**